app/src/main/kotlin/com/antosik/benchproject/app/popular/movies/entity/Movie.kt

```diff
@@ -9,9 +9,19 @@ data class Movie(
     val rating: Double,
     val releaseDate: String,
     val imageUrl: String,
-    val isFavorite: Boolean,
+    var isFavorite: Boolean,
```

Comment:
why this is a var ? in my opinion, this should be a val and we should use the advantage of data class and just use copy method when creating model with new value of isFavorite :D

Reply:
Ok, I will go with `copy` method thanks for this, I will read about it